app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -106,15 +119,26 @@ def piv_cac_enabled?
     end
 
     def process_invalid_submission
-      clear_piv_cac_information
-      flash[:error_type] = user_piv_cac_form.error_type
-      redirect_to setup_piv_cac_url
+      if user_piv_cac_form.name_taken
+        flash.now[:error] = t('errors.webauthn_setup.unique_name')
```

Comment:
Should we copy this translation to a new key. Something under `errors.piv_cac_setup.unique_name`